nbs/introduction.ipynb

```diff
@@ -89,147 +89,147 @@
      "name": "stderr",
```

Comment:
<p>In theory, we usually say that a datalog/rgxlog query is trying to find all instantiations of the free variables that satisfy the query.</p><p>And this is why if we have a query with no free variables, we get an empty set of instantiations if its true and no such set if its false.</p><p> </p><p>Perhaps you can present queries using the <code>instantiation</code> notation and then the explanation here will be easier to understand. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/134/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>